### DISCUSSION TITLE
2023-09-14

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#504] [daily_meeting]
### 오늘의 할 일
- 백엔드
  - DB 스키마 변경 논의

- 프론트엔드
  - 각자 프로젝트 진행

### 개인 일정
- 룩소(오후 근로)

### 내일의 할 일

**Comment by yeopto** [comment]
### 체크인

8점 - 피곤해서 지각했습니다.

### 체크아웃

9점 - 컨디션이 회복되어서 좋습니다.

**Reply by aak2075** [reply]
반응 뭐죠?

**Reply by jaehee329** [reply]
반응미쳐꼬

**Comment by aak2075** [comment]
### 체크인

10점 - 불목입니다👍

### 체크아웃

10점 - 재밌네요

**Reply by yeopto** [reply]
불교 단어같네요

**Comment by jaehee329** [comment]
### 체크인
8점 - 시원해서 어제 아침보다 +1

### 체크아웃
9점 - 부하테스트가 설레서 +1

**Comment by woo-jk** [comment]
### 체크인
8점 - 오늘따라 유난히 피곤하군요

### 체크아웃
8점 - 할 일을 다 못했어요

**Comment by woosung1223** [comment]
### 체크인
8점 - 오늘도 조금 피곤 하네여 ㅠㅠ

### 체크아웃
9점 - 갑자기 컨디션이 나쁘지 않네여

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 날씨도 선선해지고 미션 1단계도 일단 마무리해서 좋네요~

### 체크아웃
9점 - 피곤하긴 하지만 낼 SSE 개발 들어갈 생각하니까 좀 설레기도 하네요~

**Comment by nlom0218** [comment]
### 체크인

0.1점 - 😭

### 체크아웃

7점 - 단거 먹고 잠깐 낮잠 자니 좋네요
